conductor-client/src/main/kotlin/com/openlattice/authorization/mapstores/ExternalRolesMapstore.kt

```diff
@@ -0,0 +1,55 @@
+package com.openlattice.authorization.mapstores
+
+import com.openlattice.authorization.AccessTarget
+import com.openlattice.hazelcast.HazelcastMap
+import com.openlattice.mapstores.TestDataFactory
+import com.openlattice.postgres.PostgresArrays
+import com.openlattice.postgres.PostgresTable
+import com.openlattice.postgres.ResultSetAdapters
+import com.openlattice.postgres.mapstores.AbstractBasePostgresMapstore
+import com.zaxxer.hikari.HikariDataSource
+import org.springframework.stereotype.Component
+import java.sql.PreparedStatement
+import java.sql.ResultSet
+import java.util.*
+
+/**
+ * @author Drew Bailey (drew@openlattice.com)
+ */
+@Component
+class ExternalRolesMapstore(
+        val hds: HikariDataSource
+) : AbstractBasePostgresMapstore<AccessTarget, UUID>(
+        HazelcastMap.EXTERNAL_ROLES, PostgresTable.EXTERNAL_ROLES, hds
+) {
+    override fun generateTestKey(): AccessTarget {
+        return AccessTarget(
+                TestDataFactory.role().aclKey,
+                TestDataFactory.permissions().first()
+        )
+    }
+
+    override fun generateTestValue(): UUID {
+        return UUID.randomUUID()
+    }
+
+    override fun bind(ps: PreparedStatement, key: AccessTarget, value: UUID) {
+        var index = bind(ps, key, 1)
+        ps.setObject(index++, value)
```

Comment:
I think you need a second `ps.setObject(index++, value)` here (for the update clause)